.github/workflows/manual.yml

```diff
@@ -0,0 +1,21 @@
+name: Scheduled Merge Remote Action
```

Comment:
I don't think we want this? Might be surprising for some folks...

Reply:
unintended

Reply:
strange, don't even see it locally